build.go

```diff
@@ -13,17 +14,26 @@ func Build(logger scribe.Logger) packit.BuildFunc {
 		logger.Title("%s %s", context.BuildpackInfo.Name, context.BuildpackInfo.Version)
 
 		configParser := NewRuntimeConfigParser()
-		config, err := configParser.Parse(filepath.Join(context.WorkingDir, "*.runtimeconfig.json"))
+		var command string
+		builtAppPath := context.WorkingDir
+
+		_, publishOutputLocationSet := os.LookupEnv("PUBLISH_OUTPUT_LOCATION")
+
+		if publishOutputLocationSet {
+			builtAppPath, _ = os.LookupEnv("PUBLISH_OUTPUT_LOCATION")
+		}
```

Comment:
I think that this block is a bit cumbersome and might flow a bit nicer if you were to just do one os.LookupEnv call.